cdhweb/pages/models.py

```diff
@@ -7,7 +7,7 @@
 from django.core.exceptions import ValidationError
 from django.db import models
 from django.template.defaultfilters import striptags, truncatechars_html
-from springkit.blocks import CTABlock, FeatureBlock, ImageBlock, VideoBlock
```

Comment:
Was is the title that made this one a bust?

Reply:
yeah there is a title on the image block in springkit that we don't want in cdh